boxes/generators/paperbox.py

```diff
@@ -177,3 +254,25 @@ def tab_description(self, height, width):
             side,
             deg - 90,
         ]
+
+    def dented_tab_description(self, width, reverse=False):
+        deg = math.degrees(self.tab_angle_rad)
+        side = width / math.cos(self.tab_angle_rad)
+        end_width = width - 2 * width * math.tan(self.tab_angle_rad)
+        path = [
+            2 * self.burn,
+            -90,
+            width / 2,
+            90,
+            0,
+            (-90, width / 4),
+            width / 4,
+            90,
+            width - (width / 4) - 4 * self.burn,
+            90,
+            width,
+            -90,
+            2 * self.burn,
+        ]
```

Comment:
please try this instead:
```py
        path = [
            2 * self.burn,
            -90,
            self.lid_heigth - self.lid_radius,
            (90, self.lid_radius),
            width - self.lid_heigth - 4 * self.burn,
            90,
            self.lid_heigth,
            -90,
            2 * self.burn,
        ]
```
I tried to give the _ears_ the same length as the lid

Reply:
The point of the cut in the lid was related to the previous shape, I feel like now it can't lock sharply like it's purpose was: there is no link between the cut length and the ear, and the lid is sliding against a soft curve in the ear instead of a right angle locking it.

I would personnaly suggest to either remove the cut and keep the soft angle, or

1. use a variable to define the lid_cut_size (length/2, but maximum 1cm, and maximum a fifth of the lid) and use it both in the lid cut and the ear design
2. change a bit this design so that the cut and the ears would lock according to this common information with a right angle

What do you think ? I will push something like this to illustrate

Reply:
I also incorporated the paper thickness in the new design :-)